src/utils/normalization.ts

```diff
@@ -134,7 +132,23 @@ export const normalize = (payload: OperationsRepository) => {
     ),
   };
 
-  return { ...payload, apis, chains, dapis, explorer } as OperationsRepository; // TODO add api3 and airseeker
+  const subscriptions = payload.subscriptions
+    ? Object.fromEntries(
+        Object.entries(payload.subscriptions).map(([chain, subs]) => {
+          return [
+            chain,
+            Object.fromEntries(
+              Object.values(subs).map((subscription) => [
+                `${subscription.paymentTxHash}-${subscription.resourceId}`,
```

Comment:
Are we concatenating 2 bytes32 strings together for the file name? Wouldn't that filename be too long and not too descriptive? We don't have `resourceName` prop so I'm not sure we can even improve it if the answer to the question is yes.

Reply:
This needs to change as per the discussion yesterday:
We need to key these in separate directories by dapis and dataFeeds (dataFeedId can be a beaconId or beaconSetId).
The idea behind these file keys are that they're unique and provide a human-useful means of reference. I think if someone is searching manually for something they'll have the tx hash for a consumer subscription.

Anything more complex than that would need to be grep or similar?

Reply:
Done